src/toolchain.rs

```diff
@@ -674,6 +704,7 @@ impl<'a> Toolchain<'a> {
                     return Err(ErrorKind::UnknownComponent(
                         self.name.to_string(),
                         component.description(&manifest),
+                        self.get_component_suggestion(&component),
```

Comment:
I think it'd be better if this only suggested from the *installed* components, otherwise you might get confused by it suggesting "did you mean X" where X isn't installed.